internal/pkg/metrics/manifest.go

```diff
@@ -28,12 +37,31 @@ const (
 
 type ManifestMetrics struct {
 	*SharedMetrics
+	reconcileDurationHistogram *prometheus.HistogramVec
 }
 
 func NewManifestMetrics(sharedMetrics *SharedMetrics) *ManifestMetrics {
-	return &ManifestMetrics{SharedMetrics: sharedMetrics}
+	metrics := &ManifestMetrics{
+		SharedMetrics: sharedMetrics,
+		reconcileDurationHistogram: prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name:    metricReconcileDuration,
+				Help:    "Histogram of reconcile duration for manifest reconciliation in seconds",
+				Buckets: prometheus.DefBuckets,
```

Comment:
We might need to revisit the buckets here when the visualization is there

Reply:
aren't the buckets irrelevant if we use count and sum of the histogram metric to derive the latency visualisation? This is also how the latency metrics in current Kyma Overview dashboard are shown

Reply:
yes, there is histogram metric provided by the controller runtime. it is called `controller_runtime_reconcile_time_seconds`. This is what we are trying to replace with this metric.
We display it in Kyma Overview `Reconcile Duration` Visualization. Here is the PromQL query for it:
```
rate(controller_runtime_reconcile_time_seconds_sum{controller="manifest"}[$__rate_interval])
/
rate(controller_runtime_reconcile_time_seconds_count{controller="manifest"}[$__rate_interval])
```
Therefore, we do not need the buckets.

Reply:
but I can change the buckets to match the controller runtime metric

Reply:
okay we can have them like this for now till we have the visualization.
in the above question, I meant histograms defined in the lifecycle manager not those of the controller runtime